partner_default_sale_discount/models/sale_order.py

```diff
@@ -31,12 +31,8 @@ def fields_view_get(self, view_id=None, view_type='form', toolbar=False,
         xml_discount = eview.xpath("//field[@name='default_sale_discount']")
         if xml_order_line and xml_discount:
             # This should be handled in "string" mode
```

Comment:
Sorry, last review was on outdated changes; in any case please explain *why* here.